jetstream.md

```diff
@@ -0,0 +1,121 @@
+===
+What we're going to do here is walk through starting up a running
+computer (an "instance") on the Jetstream service. 
+
+[Jetstream](https://jetstream-cloud.org/) is an [NSF/XSEDE] resource designed to promote and provide configurable cyberinfrastructure in the form of cloud computing to both novice and experienced users.
+
+If you would like to read more about cloud computing, see this [Carpentry Cloud Computing lesson](http://www.datacarpentry.org/cloud-genomics/01-why-cloud-computing/).
+
+- This is the first and last place in these lessons where it will matter if you are using PC, Mac, or Linux. After we connect to our virtual machines built using the same image; we will all be on the same operating system/computing environment.
+
+> **NOTE: WINDOWS users** will need to install a UNIX-ready terminal to SSH-login (if you haven't already). We recommend Git bash. See workshop setup page for instructions.
+
+# Login & Launch Instance
+
+- **Login to [Jetstream](https://use.jetstream-cloud.org/application/jetstream/images/search) by clicking the "login" button towards the right-upper corner of the screen.**
+
+![](jetstream/images/login1.png)
+![](jetstream/images/login2.png)
+
+- **Fill in your Jetstream username and password and click "LOGIN"**
+
+![](jetstream/images/login3.png)
+
+- **Select the "Projects" tab and then click the "CREATE NEW PROJECT" button**
+
+![](jetstream/images/login4.png)
+
+- **Give your Project folder a name (Description is optional). Then click "CREATE".**
+
+![](jetstream/images/login5.png)
+
+- **Click on your newly created project and then Click on "NEW" and then "Instance" from the drop-down menu to start up a new virtual machine.**
+
+![](jetstream/images/login6.png)
+
+- **To select an image click on "Show All" tab and Search for "ANGUS 2019" and choose the "ANGUS 2019" image created by 'titus'.**
+
+![](jetstream/images/login7.png)
+
+Here, "image" refers to the resources that are pre-loaded into your computing workspace on your instance. Think of it like apps that come with your phone before you add new ones on your own.
+
+**You will be presented with options to choose and configure your virtual machine here:
+
+	+ Instance Name: e.x., "ANGUS 2019 base image" or you can leave it default which is the image name.
+
+	+	Base Image Version: "1.0"
+
+	+	Project: select your project folder
+
+	+	Allocation Source: DIBSI Jetstream account
+
+	+	Provider: "Jetstream - TACC"
+
+	+ Instance size: We recommend ""m1.medium" (CPU: 6, Mem: 16GB, Disk: 60GB)"for this tutorial; Though  depending on your allocations, choose most suitable one.**
+
+![](jetstream/images/login8.png)
+
+- **Launch instance and wait for the build to be deployed (~ 5-10 minutes).**
+
+> Note: During the build process: `scheduling-->building-->spawning-->deploying-->Networking-->N/A`; Be patient! Don't reload!. Once the virtual machine is ready, the "Activity" column will show "N/A" and the "Status" column will turn green and "Active".
```

Comment:
double punctuation after `reload!.`

Reply:
What does don’t reload mean, @sateeshperi? Reloading the page doesn’t affect the build